telegram/files/sticker.py

```diff
@@ -190,6 +221,10 @@ class StickerSet(TelegramObject):
         arguments had to be changed. Use keyword arguments to make sure that the arguments are
         passed correctly.
 
+    .. versionchanged:: 13.14:
+        The parameter ``contains_masks`` has been depreciated.
+        Use ``sticker_type`` instead.
```

Comment:
Maybe add something like "As of Bot API 6.2, the parameter …"? also in the other locations where this is relevant